sopel/bot.py

```diff
@@ -376,6 +376,15 @@ def reply(self, text, dest, reply_to, notice=False):
         else:
             self.say(text, dest)
 
+    def kick(self, nick, channel, text=None):
+        """Send an IRC KICK command.
+
+        Within the context of a triggered callable, ``channel`` will default to the
+        channel in which the call was triggered. If triggered from a private message,
+        ``channel`` is required (or the call to ``kick()`` will be ignored).
+        """
```

Comment:
In agreement with @half-duplex on IRC:

```suggestion

        .. versionadded:: 7.0
        """
```